src/core/Vector.rei

```diff
@@ -1,5 +1,6 @@
-[@genType]
+[@genType "Vector"]
```

Comment:
This allows you to alias how you want the type exported. However, as you can see above, it messes up generics 😭 I'll see what `@genType` has to help with this.